gtests/net/packetdrill/packetdrill.config

```diff
@@ -3,3 +3,5 @@
 # ./scripts/kconfig/merge_config.sh -m .config packetdrill.config
 
 CONFIG_TUN=y
+CONFIG_NET_SCH_FQ=y
```

Comment:
these two lines seem to be unrelated to python3... what's the motivation here? can they be a cherry-pick from some other commit? if we need them, they should be a separate commit.

Reply:
I removed this change from this patch set.
It will be needed when upload more test cases.